lib/react_on_rails/helper.rb

```diff
@@ -636,6 +657,9 @@ def server_rendered_react_component(render_options)
                                                js_code: js_code)
       end
 
+      # TODO: handle errors for rsc streams
```

Comment:
This will be fixed later on the PR that fixes
https://github.com/shakacode/react_on_rails_pro/issues/465